src/utils/osx/OSXUtils.mm

```diff
@@ -81,3 +82,22 @@
   else
     [NSCursor hide];
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////
+bool OSXUtils::isWindowFullScreen(KonvergoWindow *qtWindow)
```

Comment:
Change type to `QWindow *` here, also nust name it `window` , type makes its explicit that it's a Qt one :)